examples/vnet/doca_gw_dpdk.c

```diff
@@ -394,6 +432,10 @@ static int doca_gw_dpdk_build_item(struct doca_gw_match *match,
 			doca_gw_dpdk_build_vxlan_flow_item(NEXT_ITEM, match);
 			doca_gw_dpdk_build_inner_eth_flow_item(NEXT_ITEM, match);
 			break;
+		case DOCA_TUN_GRE:
+			doca_gw_dpdk_build_gre_flow_item(NEXT_ITEM);
```

Comment:
take next layer from here, if next proto is ipv4/ipv6

Reply:
update this methord, 
can use match->inner ipv4 or ipv6 to set ipv4/ipv6